public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx

```diff
@@ -492,9 +491,6 @@ export const QueryAndExpressionsStep = ({ editingExistingRule, onDataChange }: P
               dispatch(removeExpression(refId));
             }}
             onUpdateRefId={onUpdateRefId}
-            onUpdateExpressionType={(refId, type) => {
```

Comment:
Why do we want to remove this?

Reply:
IIRC we no longer allow updating the expression type, instead we force the user to delete and re-create an expression :)